themes/docsmith/layouts/partials/header.html

```diff
@@ -63,6 +63,7 @@
               <li class="o-menu__item"><a class="o-menu__link" href="#sub-menu--pricing"><span class="o-menu__title">Pricing</span></a></li>
               <li class="o-menu__item"><a class="o-menu__link" href="#sub-menu--community"><span class="o-menu__title">Community</span></a></li>
               <li class="o-menu__item o-menu__item--sign-up"><a class="o-menu__link" href="https://login.linode.com/signup"><span class="o-menu__title">Sign Up</span></a></li>
+              <li class="o-menu__item o-menu__item--search"><a class="o-menu__link" href="https://www.linode.com/search/"><span class="o-menu__title">Search</span></a></li>
```

Comment:
I think this was removed earlier?

Reply:
The "rogue search link" PR was never merged into the hercules repo. Merged that in, rebased the hercules PR, and rebuilt to remove it in this docs PR. Should be good now.